docs/sources/mimir/references/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Grafana Mimir references"
```

Comment:
@knylander-grafana, I could use your input about how we can standardize the `_index.md` file for `references` across the DB docs. Have thoughts on this?

Reply:
Consider these topics: 
* Introduction - section purpose with high-level description, philosophy of how things are organized (if applicable)
* Info of any major groupings of references, like developer content, configuration, architecture, etc. 

After that, have your list of {{<section>}}